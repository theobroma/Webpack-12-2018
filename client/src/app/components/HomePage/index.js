import React, { Fragment } from 'react';
import { compose } from 'recompose';

// import ProgressBar from 'components/ProgressBar';
// import Question from 'components/Question';
// import Highlight from 'components/Highlight';
// import Timer from 'components/Timer';
// import TimerM from 'components/TimerM';
// import CountDown from 'components/CountDown';

const Component = ({ someProp }) => (
  <Fragment>
    {/* <ProgressBar /> */}
    {/* <Highlight /> */}
    {/* <Question /> */}
    {/* <CountDown /> */}
    <div>Hello</div>
  </Fragment>
);

export default compose()(Component);
