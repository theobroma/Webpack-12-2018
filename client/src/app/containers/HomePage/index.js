import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';
import HomePageComponent from 'components/HomePage';
import { connect } from 'react-redux';
// import { actionName } from 'actions/action';

class HomePage extends PureComponent {
  static propTypes = {
    // getListFeedback: PropTypes.func.isRequired,
    // filterFeedback: PropTypes.func.isRequired,
  };

  // state = { checked: '0' };

  componentDidMount() {
    // this.props.getListFeedback();
  }

  handleOptionChange = e => {
    // this.setState({ checked: e.target.value });
    // this.props.filterFeedback(e.target.value);
  };

  render() {
    return (
      <HomePageComponent
        {...this.props}
        // handleOptionChange={this.handleOptionChange}
        // checked={this.state.checked}
      />
    );
  }
}

const mapStateToProps = state => ({
  // feedbacks: state.feedback.feedbacks,
  // averageRating: state.feedback.averageRating,
});

const mapDispatchToProps = {};

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(HomePage);
