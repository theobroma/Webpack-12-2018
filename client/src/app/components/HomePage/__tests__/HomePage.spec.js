import React from 'react';
import { shallow } from 'enzyme';
import HomePage from 'components/HomePage';

describe('<HomePage />', () => {
  it('HomePage', () => {
    const component = shallow(<HomePage />);

    expect(component).toMatchSnapshot();
  });
});
