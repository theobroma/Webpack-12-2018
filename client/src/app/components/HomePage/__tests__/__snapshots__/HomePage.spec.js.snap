// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> HomePage 1`] = `
<Fragment>
  <div>
    Hello
  </div>
</Fragment>
`;
